

# Allow policyException
extraArgs:
  - --loggingFormat=text
  - --exceptionNamespace={{ kyverno_namespace }}
  - --enablePolicyException=true

replicaCount: {{ kyverno_replica_count }}

image:
  repository: {{ kyverno_image_prefix }}ghcr.io/kyverno/kyverno
{% if kyverno_dockerconfigjson is defined and kyverno_dockerconfigjson %}
  pullSecrets:
    - name: image-pull
{% endif %}

initImage:
  repository: {{ kyverno_image_prefix }}ghcr.io/kyverno/kyvernopre

cleanupController:
  image:
    repository: {{ kyverno_image_prefix }}ghcr.io/kyverno/cleanup-controller
{% if kyverno_dockerconfigjson is defined and kyverno_dockerconfigjson %}
    pullSecrets:
      - name: image-pull
{% endif %}

{% if kyverno_dockerconfigjson is defined and kyverno_dockerconfigjson %}
existingImagePullSecrets:
  - image-pull
{% endif %}

# Conflicting with the webhooks config below. Aim is to also exclude kube-system
excludeKyvernoNamespace: false
config:
  webhooks:
    - namespaceSelector:
        matchExpressions:
        - key: kubernetes.io/metadata.name
          operator: NotIn
          values:
            - kube-system
            - kyverno